Resources/Prototypes/Entities/Objects/Misc/candles.yml

```diff
@@ -28,15 +28,18 @@
           volume: 10
     - type: UseDelay
     - type: Flammable
+      peakFlameTemperature: 350
       fireSpread: false
       canResistFire: false
       alwaysCombustible: true
       canExtinguish: true
-      firestacksOnIgnite: 3.0
+      firestacksOnIgnite: 3.3
       firestackFade: -0.01
       damage:
         types:
           Heat: 0.1
+    - type: Temperature
+    - type: AtmosExposed
```

Comment:
This does not need to be added here, because you `EnsureComp` it when it gets lit.

Reply:
Its not strictly necessary, and probably fine if we drop it, but it does mean that the temperature of flammable things will be wrong when they initially catch fire. Could lead to weird behavior like catching fire initially cooling the room before heating it again.

Reply:
Why is that?

This would mean that every single candle gets called in the `AtmosUpdate` event.

Reply:
They would have whatever the prototype temperature is (usually 20C) until they gain the AtmosExposedComponent and equalize. So if the room is already hot, say 50C, when the candle is lit it will initially be colder than the room and pull heat from the air until it reaches room temp.

Although now I'm saying that, I can probably just fake it by setting the entities temperature to the air temp when it first ignites.